src/vs/workbench/contrib/terminal/browser/terminalTabsList.ts

```diff
@@ -145,7 +146,8 @@ export class TerminalTabList extends WorkbenchList<ITerminalInstance> {
 		// unless multi-selection is in progress
 		this.onMouseClick(async e => {
 			if (e.browserEvent.altKey && e.element) {
-				await this._terminalService.createTerminal({ location: { parentTerminal: e.element } });
+				const cwd = await getCwdForSplit(this._terminalService.configHelper, e.element);
+				await this._terminalService.createTerminal({ location: { parentTerminal: e.element }, cwd });
```

Comment:
Was this an existing bug? We should create an issue if so, that way it'll get verified.